DirectXTex/DirectXTexWIC.cpp

```diff
@@ -588,6 +588,17 @@ namespace
                     value.bVal = 0;
                     (void)metawriter->SetMetadataByName(L"/sRGB/RenderingIntent", &value);
                 }
+                // linear space.
```

Comment:
If this is the proper behavior, I should also update the ``ScreenGrab`` modules in _DirectX Tool Kit_ for DX11 and DX12.